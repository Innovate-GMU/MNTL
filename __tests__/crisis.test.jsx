import React from 'react'
import renderer from 'react-test-renderer'
import Crisis from "pages/crisis.js"

it('renders crisis unchanged', () => {
  const tree = renderer.create(<Crisis />).toJSON()
  expect(tree).toMatchSnapshot()
})