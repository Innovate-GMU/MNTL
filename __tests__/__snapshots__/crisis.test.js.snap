// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders crisis unchanged 1`] = `
Array [
  <nav
    className="top-0 fixed z-50 w-full flex flex-wrap items-center justify-between px-2 py-3 navbar-expand-lg bg-white shadow"
  >
    <div
      className="container px-4 mx-auto flex flex-wrap items-center justify-between"
    >
      <div
        className="w-full relative flex justify-between lg:w-auto lg:static lg:block lg:justify-start"
      >
        <a
          className="text-blueGray-700 text-sm font-bold leading-relaxed inline-block mr-4 py-2 whitespace-nowrap uppercase"
          href="#pablo"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          MNTL
        </a>
        <button
          className="cursor-pointer text-xl leading-none px-3 py-1 border border-solid border-transparent rounded bg-transparent block lg:hidden outline-none focus:outline-none"
          onClick={[Function]}
          type="button"
        >
          <i
            className="fas fa-bars"
          />
        </button>
      </div>
      <div
        className="lg:flex flex-grow items-center bg-white lg:bg-opacity-0 lg:shadow-none hidden"
        id="example-navbar-warning"
      >
        <ul
          className="flex flex-col space-x-3 lg:flex-row list-none lg:ml-auto"
        >
          <li
            className="flex items-center"
          >
            <a
              href="/find-support"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              Find Support
            </a>
          </li>
          <li
            className="flex items-center"
          >
            <a
              href="/wellness-quiz"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              Take the Quiz
            </a>
          </li>
          <li
            className="flex items-center"
          >
            Your Wellness Plan
          </li>
          <li
            className="flex items-center "
          >
            <a
              href="/crisis"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              CRISIS
            </a>
          </li>
        </ul>
      </div>
    </div>
  </nav>,
  <div
    className="pt-6 pb-24 bg-gray-300"
  >
    <div>
      <div
        className="container w-24 lg:w-4/5 mx-auto mt-8 mb-24 flex flex-col"
      >
        <a
          href="https://www.fairfaxcounty.gov/police/fairfax-county-police-department"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          <div
            className="flex flex-col md:flex-row overflow-hidden bg-white rounded-lg shadow-xl  mt-4 w-100 mx-2"
            v-for="card in cards"
          >
            <div
              className="h-32 w-auto md:w-1/2"
            >
              <img
                className="inset-0 h-full w-full object-fill object-center"
                src="https://media.istockphoto.com/vectors/emergency-call-call-phone-in-hand-vector-illustration-vector-id688494412?k=20&m=688494412&s=612x612&w=0&h=zk6OfPoEbZRHpdIebIRVKww-AC2qsO8IIFtzjLlrMBs="
                style={
                  Object {
                    "height": "200px",
                    "width": "300px",
                  }
                }
              />
            </div>
            <div
              className="w-full py-4 px-6 text-gray-800 flex flex-col justify-between"
            >
              <h3
                className="text-2xl font-bold leading-normal mt-0 mb-2 text-black"
              >
                What To Do During An Emergency Situation?
              </h3>
              <p
                className="text-base font-normal leading-relaxed mt-0 mb-4 text-blueGray-800"
              >
                If you or someone you know is experiencing a mental health crisis or if you ever feel that you or someone else requires immediate medical attention:
              </p>
              <span>
                Please, Contact 
                <strong>
                  911
                </strong>
                 immediately for assistance 
              </span>
            </div>
          </div>
        </a>
        <a
          href="https://suicidepreventionlifeline.org"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          <div
            className="flex flex-col md:flex-row overflow-hidden bg-white rounded-lg shadow-xl  mt-4 w-100 mx-2"
            v-for="card in cards"
          >
            <div
              className="h-32 w-auto md:w-1/2"
            >
              <img
                className="inset-0 h-full w-full object-fill object-center"
                src="https://suicidepreventionlifeline.org/wp-content/uploads/2016/08/NSPL_Logo.jpg"
                style={
                  Object {
                    "height": "200px",
                    "width": "300px",
                  }
                }
              />
            </div>
            <div
              className="w-full py-4 px-6 text-gray-800 flex flex-col justify-between"
            >
              <h3
                className="text-2xl font-bold leading-normal mt-0 mb-2 text-black"
              >
                Suicide Prevention Information
              </h3>
              <p
                className="text-base font-normal leading-relaxed mt-0 mb-4 text-blueGray-800"
              >
                If you are actively threatening to hurt yourself, If you are thinking about suicide, or if you need emotional support, Help is available 24/7
              </p>
              <span>
                Please, contact 
                <strong>
                  800-273-8255
                </strong>
              </span>
            </div>
          </div>
        </a>
        <a
          href="https://www.crisistextline.org/"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          <div
            className="flex flex-col md:flex-row overflow-hidden bg-white rounded-lg shadow-xl  mt-4 w-100 mx-2"
            v-for="card in cards"
          >
            <div
              className="h-32 w-auto md:w-1/2"
            >
              <img
                className="inset-0 h-full w-full object-fill object-center"
                src="https://images.squarespace-cdn.com/content/v1/5c0c00c1e74940cdaa7af6a1/1550687226454-QF2ZB5MGBX5NVP7SSDCT/Crisis+text+line"
                style={
                  Object {
                    "height": "200px",
                    "width": "300px",
                  }
                }
              />
            </div>
            <div
              className="w-full py-4 px-6 text-gray-800 flex flex-col justify-between"
            >
              <h3
                className="text-2xl font-bold leading-normal mt-0 mb-2 text-black"
              >
                In A Crisis?
              </h3>
              <p
                className="text-base font-normal leading-relaxed mt-0 mb-4 text-blueGray-800"
              >
                If you ever feel like your health is worsening, If you need to talk to someone, or If you want to share your feelings with a qualified counselor
              </p>
              <span>
                Please, do not hasitate and text 
                <strong>
                  Home
                </strong>
                 to 
                <strong>
                  741-741
                </strong>
              </span>
            </div>
          </div>
        </a>
        <a
          href="https://www.rainn.org/"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          <div
            className="flex flex-col md:flex-row overflow-hidden bg-white rounded-lg shadow-xl  mt-4 w-100 mx-2"
            v-for="card in cards"
          >
            <div
              className="h-32 w-auto md:w-1/2"
            >
              <img
                className="inset-0 h-full w-full object-fill object-center"
                src="https://pbs.twimg.com/media/EPAnqiVWkAIXCoN.jpg"
                style={
                  Object {
                    "height": "200px",
                    "width": "300px",
                  }
                }
              />
            </div>
            <div
              className="w-full py-4 px-6 text-gray-800 flex flex-col justify-between"
            >
              <h3
                className="text-2xl font-bold leading-normal mt-0 mb-2 text-black"
              >
                National Sexual Assault Hotline
              </h3>
              <p
                className="text-base font-normal leading-relaxed mt-0 mb-4 text-blueGray-800"
              >
                If you or someone you know has been a victim of sexual assault, if you need Information about laws related to sex crimes in your state, or if you want to tell your story,
              </p>
              <span>
                Contact 
                <strong>
                  800-656-4673
                </strong>
                 for help and resources
              </span>
            </div>
          </div>
        </a>
      </div>
    </div>
  </div>,
  <footer
    className="relative bg-blueGray-200 pt-8 pb-6"
  >
    <div
      className="bottom-auto top-0 left-0 right-0 w-full absolute pointer-events-none overflow-hidden -mt-20 h-20"
      style={
        Object {
          "transform": "translateZ(0)",
        }
      }
    >
      <svg
        className="absolute bottom-0 overflow-hidden"
        preserveAspectRatio="none"
        version="1.1"
        viewBox="0 0 2560 100"
        x="0"
        xmlns="http://www.w3.org/2000/svg"
        y="0"
      >
        <polygon
          className="text-blueGray-200 fill-current"
          points="2560 0 2560 100 0 100"
        />
      </svg>
    </div>
    <div
      className="container mx-auto px-4"
    >
      <div
        className="flex flex-wrap text-center lg:text-left"
      >
        <div
          className="w-full lg:w-6/12 px-4"
        >
          <h4
            className="text-3xl font-semibold"
          >
            Let's keep in touch!
          </h4>
          <h5
            className="text-lg mt-0 mb-2 text-blueGray-600"
          >
            Find us on any of these platforms, we respond 1-2 business days.
          </h5>
          <div
            className="mt-6 lg:mb-0 mb-6"
          >
            <button
              className="bg-white text-lightBlue-400 shadow-lg font-normal h-10 w-10 items-center justify-center align-center rounded-full outline-none focus:outline-none mr-2"
              type="button"
            >
              <i
                className="fab fa-twitter"
              />
            </button>
            <button
              className="bg-white text-lightBlue-600 shadow-lg font-normal h-10 w-10 items-center justify-center align-center rounded-full outline-none focus:outline-none mr-2"
              type="button"
            >
              <i
                className="fab fa-facebook-square"
              />
            </button>
            <button
              className="bg-white text-pink-400 shadow-lg font-normal h-10 w-10 items-center justify-center align-center rounded-full outline-none focus:outline-none mr-2"
              type="button"
            >
              <i
                className="fab fa-dribbble"
              />
            </button>
            <button
              className="bg-white text-blueGray-800 shadow-lg font-normal h-10 w-10 items-center justify-center align-center rounded-full outline-none focus:outline-none mr-2"
              type="button"
            >
              <i
                className="fab fa-github"
              />
            </button>
          </div>
        </div>
        <div
          className="w-full lg:w-6/12 px-4"
        >
          <div
            className="flex flex-wrap items-top mb-6"
          >
            <div
              className="w-full lg:w-4/12 px-4 ml-auto"
            >
              <span
                className="block uppercase text-blueGray-500 text-sm font-semibold mb-2"
              >
                Useful Links
              </span>
              <ul
                className="list-unstyled"
              >
                <li>
                  <a
                    className="text-blueGray-600 hover:text-blueGray-800 font-semibold block pb-2 text-sm"
                    href="https://www.creative-tim.com/presentation?ref=nnjs-footer"
                  >
                    About Us
                  </a>
                </li>
                <li>
                  <a
                    className="text-blueGray-600 hover:text-blueGray-800 font-semibold block pb-2 text-sm"
                    href="https://blog.creative-tim.com?ref=nnjs-footer"
                  >
                    Blog
                  </a>
                </li>
                <li>
                  <a
                    className="text-blueGray-600 hover:text-blueGray-800 font-semibold block pb-2 text-sm"
                    href="https://www.github.com/creativetimofficial?ref=nnjs-footer"
                  >
                    Github
                  </a>
                </li>
                <li>
                  <a
                    className="text-blueGray-600 hover:text-blueGray-800 font-semibold block pb-2 text-sm"
                    href="https://www.creative-tim.com/bootstrap-themes/free?ref=nnjs-footer"
                  >
                    Free Products
                  </a>
                </li>
              </ul>
            </div>
            <div
              className="w-full lg:w-4/12 px-4"
            >
              <span
                className="block uppercase text-blueGray-500 text-sm font-semibold mb-2"
              >
                Other Resources
              </span>
              <ul
                className="list-unstyled"
              >
                <li>
                  <a
                    className="text-blueGray-600 hover:text-blueGray-800 font-semibold block pb-2 text-sm"
                    href="https://github.com/creativetimofficial/notus-nextjs/blob/main/LICENSE.md?ref=nnjs-footer"
                  >
                    MIT License
                  </a>
                </li>
                <li>
                  <a
                    className="text-blueGray-600 hover:text-blueGray-800 font-semibold block pb-2 text-sm"
                    href="https://creative-tim.com/terms?ref=nnjs-footer"
                  >
                    Terms & Conditions
                  </a>
                </li>
                <li>
                  <a
                    className="text-blueGray-600 hover:text-blueGray-800 font-semibold block pb-2 text-sm"
                    href="https://creative-tim.com/privacy?ref=nnjs-footer"
                  >
                    Privacy Policy
                  </a>
                </li>
                <li>
                  <a
                    className="text-blueGray-600 hover:text-blueGray-800 font-semibold block pb-2 text-sm"
                    href="https://creative-tim.com/contact-us?ref=nnjs-footer"
                  >
                    Contact Us
                  </a>
                </li>
              </ul>
            </div>
          </div>
        </div>
      </div>
      <hr
        className="my-6 border-blueGray-300"
      />
      <div
        className="flex flex-wrap items-center md:justify-between justify-center"
      >
        <div
          className="w-full md:w-4/12 px-4 mx-auto text-center"
        >
          <div
            className="text-sm text-blueGray-500 font-semibold py-1"
          >
            Copyright © 
            2021
             Notus NextJS by
             
            <a
              className="text-blueGray-500 hover:text-blueGray-800"
              href="https://www.creative-tim.com?ref=nnjs-footer"
            >
              Creative Tim
            </a>
            .
          </div>
        </div>
      </div>
    </div>
  </footer>,
]
`;
